tests/source/structs.rs

```diff
@@ -153,4 +153,9 @@ struct Issue677 {
 }
 
 struct Foo {}
+struct Foo {
+    }
+struct Foo {
+    // comment
+    }
```

Comment:
Could you add a test which has multiple blank lines and trailing whitespace after the comment please? Also a test which should end up on one line (I assume this means a comment with trailing whitespace, but no newline).